client/src/components/Members.js

```diff
@@ -49,17 +63,50 @@ class Members extends Component {
     });
   }
 
+  //react-select documentation https://react-select.com/props
   render() {
     return (
-      <form onSubmit={this.handleSubmit}>
-        <h1>Members Page</h1>
-        <p>Gender ID: <input type='text' name='genderID' value={this.state.genderID} onChange={this.handleChange} /></p>
-        <p>Min Age: <input type='text' name='minAge' value={this.state.minAge} onChange={this.handleChange} /></p>
-        <p>Max Age: <input type='text' name='maxAge' value={this.state.maxAge} onChange={this.handleChange} /></p>
-        <p>Family Status ID: <input type='text' name='familyStatusID' value={this.state.familyStatusID} onChange={this.handleChange} /></p>
-        <p>Max Monthly Budget: <input type='text' name='maxMonthlyBudget' value={this.state.maxMonthlyBudget} onChange={this.handleChange} /></p>
-        <input type='submit'></input>
-      </form>
+      <div>
+        <form onSubmit={this.handleSubmit}>
+          <h1>Members Page</h1>
+          <p>Gender ID: <Select
+            isMulti
+            name="genderID"
+            options={[
+              { value: 'Male', label: 'Male' },
+              { value: 'Female', label: 'Female' },
+              { value: 'Other', label: 'Other' }
+            ]}
+            value={this.state.genderID}
```

Comment:
BTW, these select options should have IDs as their values. Behind the scenes, we want to convert whatever the user chooses into IDs that will match the rows of the corresponding tables in the database.

E.g. Behind the scenes, "Male" might be represented by row id 0 in the GenderType table, for example. In the future, this db table would be used to directly generate these options, but for now we can hardcode such values. Check DBSetup.js to find the current IDs being used.